README.md

```diff
@@ -9,166 +9,107 @@ This building block integrates GitVersion ("GitVersion looks at your git history
+## Using dobi for local build
 
-In the dobi meta section of your own project you have to include all dobi files integrated in meta.yaml. This way all resources defined by bb-gitversion will be included in your project and can be executed.
+dobi should only be used via the `dobi.sh` script, because there important variables are set and the right scripts are included.
 
-To work correctly it is also necessary to adapt all paths to the new root and if necessary to make the naming of the dobi resources unique (see note).
+The following dobi resources are available:
 
-Furthermore, it is possible that you need to adapt the mount-points listed in dobi.yaml to your projects needs.
+```bash
+./dobi.sh build     #build gitversion container image
+./dobi.sh deploy    #deploy gitversion container image to registry
+ ./dobi.sh version  #generate version informations (auto called by dobi.sh
+```
 
-After that, you can use the building block to generate version informations for you:
+### Default project variables
 
-```yaml
-mount=mount-git:
-  bind: .
-  path: /git
-  read-only: false # needs to be read/write to work correctly!
+Edit `./default.env` to set default project variables.
 
-mount=mount-gen:
-  bind: ./gen/
-  path: /gen
-  read-only: false
+### Local project variables
 
-job=generate-version:
-  use: image-gitversion
-  mounts:
-  - mount-git
-  - mount-gen
-  env:
-  - "USERID={user.uid}"
-```
+If you want to override project variables, copy `./local.env.template` to `./local.env` and edit `./local.env` accordingly.<br>
+`./local.env` is ignored by git via `./.gitignore`.
 
 ## Using concourse CI for a CI/CD build
 
-The pipeline file must be uploaded to concourse CI via `fly`. 
-Enter the build users ssh private key into the file `ci/credentials.template.yaml` and rename it to `ci/credentials.yaml`. 
+The pipeline file must be uploaded to concourse CI via `fly`.
+Enter the build users ssh private key into the file `ci/credentials.template.yaml` and rename it to `ci/credentials.yaml`.
 
 **Note: `credentials.yaml` is ignored by `.gitignore` and will not be checked in.**
 
 In further releases there will be a key value store to keep track of the users credentials.
 Before setting the pipeline you might login first to your concourse instance `fly -t <target> login --concourse-url http://<concourse>:<port>`. See the [fly documentation](https://concourse-ci.org/fly.html) for more help.
 Upload the pipeline file with fly:
 
-    $ fly -t <target> set-pipeline -n -p bb-buildingblock -l ci/config.yaml -l ci/credentials.yaml -c pipeline.yaml
+    $ fly -t <target> set-pipeline -n -p bb-gitversion -l ci/config.yaml -l ci/credentials.yaml -l ci/email.yaml -c pipeline.yaml
 
 After successfully uploading the pipeline to concourse CI login and unpause it. After that the pipeline should be triggered by new commits on the master branch (or new tags if enabled in `pipeline.yaml`).
 
```

Comment:
email notification description is missing

Reply:
done

Reply:
done again ;)